### PR TITLE
Delete  GoogleUtilities/README.md

### PR DESCRIPTION
**Pull request by ncooke3**
Removes dated README in GoogleUtils directory.

### CODE REVIEW

File: GoogleUtilities/README.md [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-## Overview
-
-The GoogleUtilities pod is a set of utilities organized into CocoaPods subspecs.
-See the [podspec](../GoogleUtilities.podspec) for a summary of what utilities
-are currently included. They're used by Firebase and other Google products.
-
-Direct usage by non-Google products and CocoaPods is **NOT** currently
-recommended or supported in general.
-
-**However**, we do specifically recommend that non-Google SDKs can use the App Delegate Swizzler to hook into app delegate methods as this reduces conflicts with multiple SDKs trying to do the same.
```
Comment (paulb777) [inline GoogleUtilities/README.md:12]: @maksymmalyhin What do you think of this recommendation? Do we want to encourage this?

Comment (maksymmalyhin) [reply GoogleUtilities/README.md:12]: Probably not once we have an alternative ready. Thank you for the reminder to prioritize work on [MulticastAppDelegate](https://github.com/google/GoogleUtilities/blob/mm/app-delegate-proxy/GoogleUtilities/MulticastAppDelegate/MulticastAppDelegate.swift) for our next iterations!

**Comment by ncooke3** [comment]
@paulb777 @maksymmalyhin Is this something I should remove or should we keep it as a reminder of sorts?

**Comment by maksymmalyhin** [comment]
> @paulb777 @maksymmalyhin Is this something I should remove or should we keep it as a reminder of sorts?

I think removing the App Delegate Swizzler recommendation makes sense now. Later we will add another recommendation.

**Review by maksymmalyhin — APPROVED** [review]
LGTM

**Pull request merged**